src/VisualStudio/Core/Test/UnifiedSettings/UnifiedSettingsTests_UnifiedSettingsStorage.vb

```diff
@@ -0,0 +1,58 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis
+
+Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.UnifiedSettings
+    Partial Public Class UnifiedSettingsTests
+
+        ' Mapping from the config name to its path in Unified Settings registration file
+        Private Shared ReadOnly s_unifiedSettingsStorage As New Dictionary(Of String, UnifiedSettingsStorage)() From {
+            {"dotnet_trigger_completion_on_typing_letters", New UnifiedSettingsStorage("textEditor.%LANGUAGE%.intellisense.triggerCompletionOnTypingLetters")},
```

Comment:
💭 Is this mapping not anywhere else?

Reply:
No, not now at least.

Reply:
They might be moved to https://github.com/dotnet/roslyn/blob/33977308747bf7675931788cec11b78343edc92d/src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs#L116 in future
Unified Settings provides `Microsoft.VisualStudio.Utilities.UnifiedSettings.ISettingsManager`, which would replace the usage of all old `ISettingsManager2` `IVsFeatureFlag` and etc... but it's not required now.